sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClientIntegrationTest.java

```diff
@@ -1233,18 +1294,25 @@ private void setSenderAndReceiver(MessagingEntityType entityType, int entityInde
     }
 
     private void setReceiver(MessagingEntityType entityType, int entityIndex, boolean isSessionEnabled) {
+        setReceiver(entityType, entityIndex, isSessionEnabled, Duration.ZERO);
+    }
+
+    private void setReceiver(MessagingEntityType entityType, int entityIndex, boolean isSessionEnabled,
```

Comment:
Instead of adding another overload, just create the builder client in the method. This is only used in two methods. Or make something more generic.